styles/state.css

```diff
@@ -11,16 +11,34 @@
     display: flex;
     flex-direction: row;
     justify-content: space-around;
+    text-align: center;
   }
 
   .navigation:hover ul {
```

Comment:
On hover is great for PCs since we have mice. Though, you can't hover on a cell phone. Since you're designing mobile first you should change on hovers to on click.